flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -469,6 +479,73 @@ void TizenWindowEcoreWl2::Show() {
   ecore_wl2_window_show(ecore_wl2_window_);
 }
 
+void TizenWindowEcoreWl2::UpdateFlutterCursor(const std::string& cursor_name) {
+#ifdef TV_PROFILE
+  int pointer_size = -1;
+  if (vconf_get_int(kSysMouseCursorPointerSizeVConfKey, &pointer_size) < 0) {
```

Comment:
pointer_size will be 1.
If there is an option to change the mouse pointer size in options of the device, you can change this value.

Reply:
Can I configure this value in the system settings?

Reply:
If there is no size option in System setting, you can edit vconf manually.
`vconftool set -f -t int db/menu/system/mouse-pointer-size 1`